weathersimulator-python/func.yaml

```diff
@@ -0,0 +1,18 @@
+name: weathersimulator-python
```

Comment:
Are we planing on deploying this app as a function? Because the current code is not compatible with serverless functions.

Reply:
Pablo, so yes I want to deploy it as a serverless function.
Probably will connect with you and see what changes we can do in the app so that we can deploy it as a function.